plugin/input/http/README.md

```diff
@@ -22,5 +22,39 @@ Which protocol to emulate.
 
 <br>
 
+**Example:**
+Emulating elastic through http:
+```yaml
+pipelines:
+  example_k8s_pipeline:
+    settings:
+      capacity: 1024
+    input:
+      # define input type.
+      type: http
+      # pretend elastic search, emulate it's protocol.
+      emulate_mode: elasticsearch
+      # define http port.
+      address: ":9200"
+    actions:
+      # parse elastic search query.
+      - type: parse_es
+      # decode elastic search json.
+      - type: json_decode
+        # field is required.
+        field: message
+    output:
+      # Let's write to kafka example.
+      type: kafka
+        brokers: [kafka-local:9092, kafka-local:9091]
+        default_topic: yourtopic-k8s-data
+        use_topic_field: true
+        topic_field: pipeline_kafka_topic
+      
+      # Or we can write to file: 
+      # type: file
+      # target_file: "./output.txt"
+```
+
```

Comment:
A couple of test queries also be helpful here

Reply:
I added example